ci-operator/config/openshift-kni/cluster-group-upgrades-operator/openshift-kni-cluster-group-upgrades-operator-release-4.11.yaml

```diff
@@ -0,0 +1,83 @@
+base_images:
+  base:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  to: cluster-group-upgrades-operator
+- dockerfile_path: bundle.Dockerfile
+  to: cluster-group-upgrades-operator-bundle
+- dockerfile_path: Dockerfile.precache
+  from: base
+  to: cluster-group-upgrades-operator-precache
+- dockerfile_path: Dockerfile.recovery
+  from: base
+  to: cluster-group-upgrades-operator-recovery
+operator:
+  bundles:
+  - dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/openshift-kni/cluster-group-upgrades-operator:.*
+    with: pipeline:cluster-group-upgrades-operator
+promotion:
```

Comment:
missing `disabled: true` under promotion config

Reply:
After that, this should work and match the different openshift configs for the release.